apps/lead/dataloaders.py

```diff
@@ -103,6 +104,17 @@ def batch_load_fn(self, keys):
         return Promise.resolve([_map.get(key) for key in keys])
 
 
+class LeadDraftEntryDiscardCountLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        draft_entry_qs = DraftEntry.objects.filter(lead__in=keys, is_discarded=True).annotate(count = models.Count('id'))
+        _map = {
```

Comment:
We can use discarded as value and then do group by it, resulting in True/False counts.